test/videojs.record.spec.js

```diff
@@ -186,6 +186,48 @@ describe('Record', () => {
         });
     });
 
+    /** @test {Record} */
+    it('runs as audio-screen plugin', (done) => {
+        // create audio-screen plugin
+        player = TestHelpers.makeAudioScreenPlayer();
+
+        // correct device button icon
+        expect(player.deviceButton.buildCSSClass().endsWith(
+            'sv-perm')).toBeTrue();
+
+        let browser = detectBrowser();
+        if (isFirefox() || (browser.browser === 'chrome' && browser.version >= 72) ||
```

Comment:
i think this browser check statement is also a copy/paste and should be refactored so it can be reused.

Reply:
163d07c